pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -407,3 +416,68 @@ def from_pymatgen(pymatgen_obj):
     @wraps(ovito_to_pyiron)
     def from_ovito(ovito_obj):
         return ovito_to_pyiron(ovito_obj)
+
+    def get_high_index_surface(self, element='Ni', crystal_structure='fcc', lattice_constant=3.526,
```

Comment:
I agree with @liamhuber. For consistency, the name `high_index_surface` would be a better name.